fs/mqueue/Kconfig

```
@@ -11,4 +11,10 @@ config FS_MQUEUE_MPATH
 	---help---
 		The path to where POSIX message queues will exist in the VFS namespace.
 
+config FS_MQUEUE_NPOLLWAITERS
+	int "mesage queue number of poll waiters"
+	default 4
+	---help---
+		Mesage queue number of poll waiters
```

Please, here is a suggestion to improve the description.
```suggestion
config FS_MQUEUE_NPOLLWAITERS
	int "Maximum number of poll waiters"
	default 4
	---help---
		The maximum number of waiters for the poll operation.
```